src/network/hooks/useSponsorshipsList.ts

```diff
@@ -13,19 +13,25 @@ import { getAllSponsorships, getSponsorshipsByCreator } from '~/getters'
 export const useAllSponsorshipsQuery = (
     pageSize = 10,
     searchQuery?: string,
-): UseInfiniteQueryResult<SponsorshipElement[]> => {
+): UseInfiniteQueryResult<{ pageParam: number; elements: SponsorshipElement[] }> => {
```

Comment:
`pageParam` → `page`?

Reply:
yeah - kinda got too inspired by the tansack query's property names :D I'll change it to what it actually is - `skippedElements`

Reply:
It'd a number of skipped elements. 😃